build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -942,6 +950,25 @@ message ExecutedActionMetadata {
   // When the worker completed executing the action command.
   google.protobuf.Timestamp execution_completed_timestamp = 8;
 
+  // New in v2.3: the amount of time the worker spent executing the action
+  // command.
+  //
+  // The execution duration is only intended to cover the "execution" of the
+  // specified action and not time in queue nor any overheads before or after
+  // execution such as marshalling inputs/outputs. The server SHOULD avoid
+  // including time spent the client doesn't have control over, and MAY extend
+  // or reduce the execution duration to account for delays or speedups that
+  // occur during execution itself (e.g., lazily loading data from the Content
+  // Addressable Storage, live migration of virtual machines, emulation
+  // overhead).
+  //
+  // The method of timekeeping used to compute the execution duration MUST be
+  // consistent with what is used to enforce the
+  // [Action][[build.bazel.remote.execution.v2.Action]'s `timeout`. There is no
+  // arithmetical relationship between the execution duration and the values of
```

Comment:
nitpick: the last sentence is a bit difficult to read. How about something like "The difference between execution_start_timestamp and execution_completed_timestamp may be greater than the specified timeout, to account for non-execution time." ?

Reply:
But it may also be less. There simply is no relationship.

Reply:
Right- you're pointing out that it can also be greater due to overheads/non-execution times (and that's the part to emphasise).

Reply:
Would s/arithmetical/specific/ do?

Reply:
Or even just s/arithmetical// ?

Reply:
Removed arithmetical